### DISCUSSION TITLE
The fix to put decorations outside

### DISCUSSION THREAD

**Discussion by arandomuser0** [winstxnhdw/lc-hax#463] [Ideas]
https://thunderstore.io/c/lethal-company/p/Vegasx/Build_Anywhere/source/

**Comment by winstxnhdw** [comment]
It’s host only.

**Comment by winstxnhdw** [comment]
The author claims it works server-side though. Can someone test this as non-host? Make sure others are able to see the change in furniture too. Also, make sure to test on only teleporters. I know certain furnitures can still build outside, like pumpkin, but only because it’s a bug, and I don’t support bugs because they’ll likely be fixed.

**Reply by joep26020** [reply]
ill try it out

**Reply by joep26020** [reply]
can u help me fix my launch error real quick I can't use the software

**Reply by joep26020** [reply]
tested it. seems to only be outside ship on your side. other peoples side it only moves when u move it around in ship. could be somewhat useful

**Reply by winstxnhdw** [reply]
Figures.
